app/src/main/java/com/appdev/eateryblueandroid/ui/components/home/EateryFilters.kt

```diff
@@ -1,53 +1,71 @@
 package com.appdev.eateryblueandroid.ui.components.home
 
 import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
 import androidx.compose.foundation.layout.PaddingValues
 import androidx.compose.foundation.layout.Row
 import androidx.compose.foundation.layout.padding
 import androidx.compose.foundation.lazy.LazyRow
 import androidx.compose.foundation.lazy.items
 import androidx.compose.foundation.shape.RoundedCornerShape
 import androidx.compose.material.Surface
-import androidx.compose.runtime.Composable
+import androidx.compose.runtime.*
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.res.colorResource
 import androidx.compose.ui.unit.dp
 import com.appdev.eateryblueandroid.R
 import com.appdev.eateryblueandroid.ui.components.core.Text
 import com.appdev.eateryblueandroid.ui.components.core.TextStyle
 
 @Composable
-fun EateryFilters() {
-    val options = listOf(
+fun EateryFilters(alreadySelected: List<String>, onFiltersChange: (updated: List<String>) -> Unit) {
+    var selected by remember { mutableStateOf(listOf<String>()) }
+    val campusLocations = listOf("North", "West", "Central")
+    val options = mutableListOf(
```

Comment:
can we add filters as a field in HomeViewModel? this way all state is in the same place and we won't need to add the refreshMainItems function and pass that around bc it will auto update when the viewmodel data changes